src/community/json-ld/src/main/java/org/geoserver/jsonld/JsonLdGenerator.java

```diff
@@ -117,7 +117,7 @@ public void writeValueNode(String entryName, JsonNode valueNode) throws IOExcept
      * @throws IOException
      */
     public void writeResult(Object result) throws IOException {
-        if (result instanceof String || result instanceof Number) {
+        if (result instanceof String || result instanceof Number || result instanceof Boolean) {
```

Comment:
Bellow we have:
```
} else {
    writeNull();
}
```
I would recommend:
```
} else if (result == null) {
    writeNull();
} else {
    throw new RuntimeException("We can't handle the provided object o type '" + result.getClass() + "'."):   
}
```
... this will make the method more resilient, we write NULL when the object is indeed NULL, if we we don't handle the object then we throw an exception.

Reply:
fixed